stdlib/InteractiveUtils/src/macros.jl

```diff
@@ -158,13 +187,15 @@ function merge_namedtuple_types(nt::Type{<:NamedTuple}, nts::Type{<:NamedTuple}.
     NamedTuple{Tuple(names), Tuple{types...}}
 end
 
-function gen_call_with_extracted_types(__module__, fcn, ex0, kws=Expr[])
+default_is_code_macro(fcn) = startswith(string(fcn), "code_")
+
+function gen_call_with_extracted_types(__module__, fcn, ex0, kws=Expr[]; is_code_macro = default_is_code_macro(fcn))
     if isexpr(ex0, :ref)
         ex0 = replace_ref_begin_end!(ex0)
     end
     # assignments get bypassed: @edit a = f(x) <=> @edit f(x)
     if isa(ex0, Expr) && ex0.head == :(=) && isa(ex0.args[1], Symbol) && isempty(kws)
-        return gen_call_with_extracted_types(__module__, fcn, ex0.args[2])
+        return gen_call_with_extracted_types(__module__, fcn, ex0.args[2], Expr[]; is_code_macro)
```

Comment:
```suggestion
        return gen_call_with_extracted_types(__module__, fcn, ex0.args[2], kws; is_code_macro)
```

Not a big deal, but saves an allocation